src/main/java/jeju/controller/MemberController.java

```diff
@@ -98,4 +101,23 @@ public String signin(@ModelAttribute MemberTableDto dto)
 		return map;
 	}
 	
+	@PostMapping("/member/login/findpass/check")
+	public String findapss(MemberTableDto dto, Model model, @RequestParam String id,
+			@RequestParam String name, @RequestParam Date birth)
```

Comment:
함수 이름에 오타가 나서, 수정해주면 좋을 것 같아요! 어차피 함수명은 mapping에선 큰 의미 없으니까 이부분만 commit 다시하시면 좋을 것 같습니다!
```suggestion
	public String findpass(MemberTableDto dto, Model model, @RequestParam String id,
			@RequestParam String name, @RequestParam Date birth)
```